### DISCUSSION TITLE
Create a guide for noobs

### DISCUSSION THREAD

**Discussion by brokeboiflex** [oobabooga/text-generation-webui#2162] [Ideas]
At the time of writing this there are **479** open issues and only a part of them are related to _text-generation-webui_ rest is related to CUDA installations, conda, python etc . It makes it hard both for the maintainers and the users to process the issues. 

I suggest documenting the "scope" of this repo, pointing out where users should search for their issues which are not directly related to oobabooga

**Comment by miles-du** [comment]
I agree. As a noob, I am unsure what package is the one that really relates to the issue I'm having. A diagram / map of which package does what in the structure would be a great start.

**Comment by luthis1124** [comment]
I really want to do this. I'm in the middle of figuring out what's going on so I can put together a proper wiki. With a good wiki we will see some really kickass extensions

**Reply by miles-du** [reply]
Excellent! Would be an important contribution.
